blocks/columns/columns.css

```diff
@@ -3,6 +3,20 @@
     flex-direction: column;
     }
 
+.columns-container h2 {
+    margin: var(--spacer-element-07) 0;
+    }
+
+/* this is considered an eyebrow style */
+.columns-container h4 {
+    font-family: var(--sans-serif-font-medium);
+    font-size: var(--font-size-11);
```

Comment:
Does this font size need to change based on screen width?

Reply:
Same Q here. Every `h4` should get this style if the section also contains a columns block?